src/Components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Banner from "../Banner/Banner";
+import LessProductsGrid from "../LessProductsGrid/LessProductsGrid";
+import { landingPageData } from "../../data/faker-landingPageData";
+import Footer from "../Footer/Footer";
+import { useNavigate } from "react-router-dom";
+
+function LandingPage() {
+  let navigate = useNavigate();
```

Comment:
Since the value of navigate would be constant It would be preferable to have it as a const.